apps/accounts/models/hosting.py

```diff
@@ -47,10 +48,40 @@ class Datacenter(models.Model):
     temperature_type = models.CharField(
         max_length=255, choices=TempType.choices, db_column="temperaturetype"
     )
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True
+    )
     virtual = models.BooleanField()
     website = models.CharField(max_length=255)
 
+    @property
+    def users(self) -> models.QuerySet["User"]:
```

Comment:
Thank you for adding the type hints 💚 

It would be wonderful if this become the norm, but until we're using python 3.11 or 3.12 across the board, it's a "nice to see", not a hard requirement.